Modules/_operator.c

```diff
@@ -1554,24 +1621,6 @@ methodcaller_traverse(methodcallerobject *mc, visitproc visit, void *arg)
     return 0;
 }
 
-static PyObject *
-methodcaller_call(methodcallerobject *mc, PyObject *args, PyObject *kw)
-{
-    PyObject *method, *obj, *result;
```

Comment:
`methodcaller_call()` will be almost unused ?
If so, I prefer to keep the original unchanged  for maintenability.

Reply:
It's not changed at all, just moved next to methodcaller_vectorcall as I think that makes things easier to track (and then both have to come before methodcaller_new, or else we'd need a forward declaration; I can switch to that if you prefer).

Reply:
Would it be possible for `methodcaller_call` to rollback to the previous version implemented with `PyObject_Call`  ?  Any position seems fine.
It is over 5 years old (well tested), and sufficient for a fallback function.

Reply:
Ah, I see what you mean, sorry I forgot the change.  I just put the whole thing back where it was and with the old version, to decrease the diff.